src/v2/Artsy/Relay/middleware/metaphysicsExtensionsLoggerMiddleware.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Likewise, perhaps `group` and `groupCollapsed` could also benefit from being allowed. They're pretty rarely used, so it seems safe enough.